understanding_questions

```diff
@@ -0,0 +1,50 @@
+1.  model.py is the file how define objects in db table
+
+2.  views.py is functions that handles the client requests
+
+3.  url.py map th urls to the view function
+
+4.  is the way to make changes in the models add field deleting .. into the database 
+we change the to the django model 
+then create migration we use this commend python manage.py makemigrations
+this commend analyze the changes and create new migration 
+to apply we use this commend python manage.py migrate and we done 
+
+5. serializer is converting objects to javascript data and frameworkes 
+
+6. is a toolkit the reduces the code you need to write to create  rest 
+
+7. there is no file
```

Comment:
Please search about them in google. We'll check also why there was no file.
Serializers are crucial - so please make sure you understand them.